app/dts/bindings/behaviors/zmk,behavior-sensor-rotate.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2020 The ZMK Contributors
```

Comment:
Copyright headers should be updated to the current year (2022)

Reply:
Whoops! Fixed these.